sphinxval/sphinx/sphinx.py

```diff
@@ -64,11 +64,17 @@ def validate(data_list, model_list, top=None, DoResume=False, df_pkl=""):
             
     """
     setup_logging()
+    
+    #Reconstruct command line execution of sphinx
```

Comment:
This should be done much more simply with sys.argv https://docs.python.org/3/library/sys.html#sys.argv

Reply:
You don't need to keep track of all your own arguments.

Reply:
logger.info("SPHINX called with:",  sys.argv) did not work. It passed a logging error:
 File "/Users/kwhitman/Documents/Programs/Validation/SPHINX/sphinxval/sphinxval/sphinx/sphinx.py", line 66, in validate
    logger.info("SPHINX called with:", sys.argv)
Message: 'SPHINX called with:'
Arguments: (['bin/sphinx.py', '--ModelList', 'model/SEPVAL2023_Europe/SEPVAL2023_Europe_all_20240806_NoAMSU_2.list', '--DataList', 'lists/obs_GOES_SEPVAL2023.list', '--Resume', 'output/pkl/SPHINX_dataframe.pkl'],)

Also, the logger didn't like the comma separated inputs. I had to convert to a string. But having the sys.argv variable made it much easier!